packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -31,16 +31,18 @@ export class Checkboxes extends GOVUKFrontendComponent {
     super()
 
     if (!($module instanceof HTMLElement)) {
-      throw new ElementError(`[data-module="${Checkboxes.moduleName}"]`, {
+      throw new ElementError({
         componentName: 'Checkboxes',
-        element: $module
+        element: $module,
+        identifier: `Root element (\`$module\`)`
       })
     }
 
     const $inputs = $module.querySelectorAll('input[type="checkbox"]')
     if (!$inputs.length) {
-      throw new ElementError('input[type="checkbox"]', {
-        componentName: 'Checkboxes'
+      throw new ElementError({
+        componentName: 'Checkboxes',
+        identifier: 'Input elements <input type="checkbox">'
```

Comment:
Just putting this out there, but we say "Form field" when it's of unknown type

Should we say "Form input" when we know it's an input?

```suggestion
        identifier: 'Form input (\`<input type="checkbox">\`)'
```

Reply:
Ah it's plural, so `Form inputs`?